source/adapter.screenshare.js

```diff
@@ -0,0 +1,161 @@
+(function () {
+
+  'use strict';
+
+  var tempGetUserMedia = null;
+
+  if (window.navigator.mozGetUserMedia) {
+    tempGetUserMedia = window.navigator.getUserMedia;
+
+    window.navigator.getUserMedia = function (constraints, successCb, failureCb) {
+      constraints = constraints || {};
+
+      if (constraints.video ? !!constraints.video.mediaSource : false) {
```

Comment:
we orientate ourselves along firefox syntax where you do mediaSource = 'window'. that's why
(See http://hancke.name/webrtc/screenshare/#/28)